x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/extractor/ExtractedFieldsTests.java

```diff
@@ -32,7 +32,7 @@ public void testAllTypesOfFields() {
         ExtractedField sourceField1 = new SourceField("src1", Collections.singleton("text"));
         ExtractedField sourceField2 = new SourceField("src2", Collections.singleton("text"));
         ExtractedFields extractedFields = new ExtractedFields(Arrays.asList(
-                docValue1, docValue2, scriptField1, scriptField2, sourceField1, sourceField2));
+                docValue1, docValue2, scriptField1, scriptField2, sourceField1, sourceField2), Collections.emptyMap());
```

Comment:
Should there be at least one unit test with non-empty map provided to this constructor?

Reply:
There is in `AnalyticsProcessConfigTests` where this is tested. Testing it here would just test the getter.